.eslintrc.cjs

```diff
@@ -35,6 +35,20 @@ module.exports = {
       'rules': {
         '@typescript-eslint/no-non-null-assertion': 2,
         '@typescript-eslint/no-explicit-any': 2,
+        '@typescript-eslint/member-delimiter-style': ['error', {
+          'multiline': {
+            'delimiter': 'semi',
+            'requireLast': true,
+          },
+          'singleline': {
+            'delimiter': 'semi',
+            'requireLast': true,
+          },
+        }],
+        'object-shorthand': 'error',
```

Comment:
I'm kind of indifferent to this rule, but it looks like it's `--fix`able so, :+1: